moshi/src/test/java/com/squareup/moshi/ClassJsonAdapterTest.java

```diff
@@ -25,7 +25,6 @@
 import org.junit.Test;
 
 import static com.squareup.moshi.TestUtil.newReader;
-import static com.squareup.moshi.Util.NO_ANNOTATIONS;
```

Comment:
Good call. I originally replaced it with a bunch of `Set`s, and when I didn't like that I unintentionally changed the imports.